app/views/common_parameters/welcome.html.erb

```diff
@@ -0,0 +1,12 @@
+<% content_for(:title, _("Global Parameters")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("list-ul", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Global Parameters') %></h1>
+  <p><%= _("Foreman’s concept of parameters maps onto Puppet’s idea of default-scope parameters. Foreman allows us to define a hierarchy of parameter inheritance, where global parameters global parameters accessible from any manifest") %></p>
```

Comment:
1. https://github.com/theforeman/foreman/pull/4475#discussion_r112647532 suggested removing the Puppet references, or at least emphasising their non-Puppet use in templates etc. I don't think listing Puppet first and "manifests" (Puppet manifests?) is useful to users.
1. "global parameters global parameters" twice.
1. Missing full stop.
1. Reintroduced unicode apostrophe marks, use regular single quotes.